scopes/react/react/jest/transformer.js

```diff
@@ -1,8 +1,26 @@
 /* eslint-disable global-require */
 const { transform } = require('@babel/core');
 
+const hasJsxRuntime = (() => {
+  if (process.env.DISABLE_NEW_JSX_TRANSFORM === 'true') {
+    return false;
+  }
+
+  try {
+    require.resolve('react/jsx-runtime');
```

Comment:
The `react/jsx-runtime` will always be resolved from the react env context, so no necessary to check it.